### PR TITLE
Remove public key cache

### PR DESCRIPTION
**Pull request by dlongley**
Tests passing, ready for review.

### CODE REVIEW

**Comment by dlongley** [comment]
This is ready for review.

File: lib/signature.js [hunk 1]
```diff
@@ -44,131 +39,38 @@ api.verify = callbackify(async ({doc, ledgerNodeId}) => {
...
-          publicKey: publicKeyId,
-          publicKeyBase58: _getPublicKeyFromId(publicKeyId)
-        };
-        callback(null, newKey);
-      }]
-    }, (err, results) => callback(err, results.getKey));
+  async function _getPublicKey(publicKeyId, options) {
+    publicKeyBase58 = _voters.getPublicKeyFromId({voterId: publicKeyId});
+    return {
+      '@context': 'https://w3id.org/security/v2',
```
Comment (mattcollier) [inline lib/signature.js]: constants.SECURITY_CONTEXT_V2_URL

File: lib/signature.js [hunk 2]
```diff
@@ -44,131 +39,38 @@ api.verify = callbackify(async ({doc, ledgerNodeId}) => {
...
-      };
-      return callback(null, keyOwner);
-    }
-    // TODO: don't need cache, always generate the owner document
-    // cache miss, create the owner document
-    keyOwner = {
-      '@context': 'https://w3id.org/webledger/v1',
+  async function _getPublicKeyOwner(owner, options) {
+    return keyOwner = {
+      '@context': 'https://w3id.org/security/v2',
```
Comment (mattcollier) [inline lib/signature.js]: constants.SECURITY_CONTEXT_V2_URL

File: lib/voters.js [hunk 3]
```diff
@@ -147,16 +328,9 @@ async function _createVoter(ledgerNodeId) {
   const voter = {
     id: voterId,
     ledgerNodeId: ledgerNodeId,
```
Comment (mattcollier) [inline lib/voters.js]: just `ledgerNodeId`

**Pull request merged**